client/components/spinner-button/README.md

```diff
@@ -0,0 +1,38 @@
+SpinnerButton
+===========
+
+`<SpinnerButton />` is a React component for rendering a button with an attached loading indicator.
+
+## Usage
+
+```jsx
+import SpinnerButton from 'components/spinner-button';
+
+export default function MyComponent() {
+	return (<SpinnerButton
```

Comment:
Formatting here is a bit odd (in fact, invalid since there's no closing paranthesis). I'd have expected:

```js
return <SpinnerButton />;
```